### PR TITLE
Add failing tests for rendering with layouts

### PR DESCRIPTION
**Pull request by fsateler**
### Summary

This is a merge request for discussing a possible API for implementing #391 

I would have liked to have layouts included by default with `render`. I would understand if that is deemed an API break and a alternative needs to be determined.

Additionally, I have added a test that checks the view context is shared between the layout and the component.

### CODE REVIEW

**Comment by joelhawksley** [comment]
@fsateler would you be up for pairing on this one? Email me: joelhawksley@github.com

**Comment by joelhawksley** [comment]
@fsateler would you be up for updating this PR?

File: test/sandbox/app/controllers/layouts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+# Tests for render with/without layouts
+class LayoutsController < ActionController::Base
+  layout "global_for_action", only: :global_for_action
```
Comment (fsateler) [inline test/sandbox/app/controllers/layouts_controller.rb:5]: For some reason `"application"` layout is not searched... maybe this is new in rails, and we need an ApplicationController?

Comment (Spone) [reply test/sandbox/app/controllers/layouts_controller.rb:5]: You were right :) Adding an `ApplicationController` makes the tests pass!

See https://api.rubyonrails.org/classes/ActionView/Layouts.html#module-ActionView::Layouts-label-Layout+assignment

**Comment by Spone** [comment]
Can we merge this @joelhawksley?

**Pull request merged**